sdcm/sct_config.py

```diff
@@ -171,7 +171,17 @@ class SCTConfiguration(dict):
                   Test duration (min). Parameter used to keep instances produced by tests
                   and for jenkins pipeline timeout and TimoutThread.
              """),
-
+        dict(name="prepare_duration", env="SCT_PREPARE_DURATION", type=int,
+             help="""
+                  Time in minutes, which is required to run prepare stress commands
+                  defined in prepare_*_cmd for dataset generation, and is used in
+                  test duration calculation
+             """),
+        dict(name="stress_duration", env="SCT_STRESS_DURATION", type=int,
+             help="""
+                  Time in minutes, Time of execution for stress commands from stress_cmd parameters
+                  and is used in test duration calculation
```

Comment:
same here, will it be used to `test duration calculation` only?

Reply:
change description. is it clear now?

Reply:
yes